### PR TITLE
feat: add button to copy content from dashboard terminal

### LINKED ISSUE #2587: Add button to copy build-log content to clipboard
**What would you like to be added**:
A button to copy all current content of the build-log into the clipboard

**Why would you like this to be added**:
cc @humphd

### PR DESCRIPTION
**Pull request by Andrewnt219**
## Issue This PR Addresses
Closes #2587 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Add a button to copy all the current content of the terminal to user's clipboard.

https://www.awesomescreenshot.com/video/6482705?key=d5fca93e575c13972ea6e453729c0817

## Test plan

- Add this line to `src/api/status/public/js/build-log/terminal.js`

```js
terminal.write(`Lorem ipsum dolor sit amet consectetur adipisicing elit. Quia consectetur nihil dolore accusamus accusantium eius tempora, magnam, error inventore consequuntur quisquam, at unde fuga. Natus deleniti minus animi iure laudantium?
A


quae libero

inventore provident


quisquam,

suscipit corporis odit vero consequatur doloremque nam! Ea iusto deserunt fugit rem debitis, vero obcaecati, magni saepe ipsum incidunt ex, ad maiores unde asperiores.
Nobis quis a doloribus odit nulla aliquid sapiente velit, consequuntur molestiae illo dolore tempore accusantium labore asperiores alias aspernatur iste? Assumenda impedit omnis quasi animi repudiandae ab nobis a velit?
`);
```
- Run the project, go to /build.html. 
- Click button "Copy to clipboard"
- The content of the terminal should be copied

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by humphd — CHANGES_REQUESTED** [review]
I love this.  Let's iterate on the UI a bit.  I think losing all that horizontal space across the bottom is not worth it.  Can we integrate the button into the build header?  Or could we make it a float over the top-right of the terminal somehow?

**Comment by Andrewnt219** [comment]
Let's go with the top-right. I can add a bit more padding-right to the terminal and just put a copy icon there.

**Comment by Andrewnt219** [comment]
PR description is updated with new UI.

File: src/api/status/public/css/terminal.css [hunk 1]
```diff
@@ -1,4 +1,23 @@
 .xterm {
   padding: 0.5rem;
   height: 89vh;
+  /* padding-right: 8rem;
```
Comment (humphd) [inline src/api/status/public/css/terminal.css:4]: Let's not do this.  If you don't need it, remove it.

**Review by humphd — CHANGES_REQUESTED** [review]
Really nice.  One nit.

File: src/api/status/public/css/terminal.css (hunk 1 above)
Comment (Andrewnt219) [reply src/api/status/public/css/terminal.css:4]: Oops, how did I miss that.

**Review by humphd — APPROVED** [review]
Lovely

**Review by Kevan-Y — APPROVED** [review]
LGTM, tested locally, very nice feature!

**Pull request merged**